cluster/manifests/skipper/deployment.yaml

```diff
@@ -80,6 +80,7 @@ spec:
           - "-api-usage-monitoring-realm-keys=https://identity.zalando.com/realm"
           - "-api-usage-monitoring-client-keys=https://identity.zalando.com/managed-id,sub"
           - "-api-usage-monitoring-default-client-tracking-pattern=services[.].*"
+          - "-default-filters-dir=/etc/config/default-filters"
```

Comment:
Will this work if the file doesn't exist?

Reply:
It's a directory and it is in the volume and yes it is checked before they open files.

Reply:
I see it's behind the if guard, missed it in the diff view :)